discord/install.sh

```diff
@@ -8,17 +8,18 @@ case "$1" in
 	echo "Downloading SinusBot..."
 	curl -s https://www.sinusbot.com/pre/sinusbot-0.14.3-0e747fd.tar.bz2 | tar xj
 	chmod 755 sinusbot
-	mv scripts default_scripts
+	mkdir default_scripts
+	mv scripts/* default_scripts
```

Comment:
Is there a reason for this? Should work fine the way it was.

Reply:
Nope, no clue why I changed this.